ci/Dockerfile-envoy

```diff
@@ -1,17 +1,16 @@
 ARG BUILD_OS=ubuntu
 ARG BUILD_TAG=18.04
 
-# Final stage
 FROM $BUILD_OS:$BUILD_TAG
 ARG TARGETPLATFORM
 
-RUN apt-get update && apt-get upgrade -y \
-    && apt-get install --no-install-recommends -y ca-certificates \
-    && apt-get autoremove -y && apt-get clean \
+RUN apt-get update \
+    && apt-get upgrade -qq -y \
+    && apt-get autoremove \
+    && apt-get clean \
     && rm -rf /tmp/* /var/tmp/* \
-    && rm -rf /var/lib/apt/lists/*
-
-RUN mkdir -p /etc/envoy
+    && rm -rf /var/lib/apt/lists/* \
+    && mkdir -p /etc/envoy
```

Comment:
Not a big deal but, is this required? I think when we do `ADD configs/envoyproxy_io_proxy.yaml /etc/envoy/envoy.yaml` the dir will be automatically created.

Reply:
yep, i can remove